src/suite-design-guide/general-principles.rst

```diff
@@ -433,13 +433,13 @@ Sharing by inheritance of task families is recommended when more than a few
 configuration items are involved.
 
 The simplest application of inheritance is to set global defaults in the
-``[[runtime]][root]`` namespace that is inherited by all tasks.
+``[runtime][root]`` namespace that is inherited by all tasks.
```

Comment:
Should this be 
```suggestion
``[runtime][[root]]`` namespace that is inherited by all tasks.
```

Reply:
I thought when writing a section/setting out "flat" we're supposed to ignore the multiple brackets?

Reply:
https://cylc.github.io/cylc-doc/8.0a2/html/tutorial/scheduling/graphing.html#shorthand

Reply:
Ah pass, I've not come across that yet but it is entirely possible.

Reply:
Yes, if you look at e.g. https://cylc.github.io/cylc-doc/8.0a2/html/reference/config/suite.html#suite.rc[meta]URL it's also how you write rst references
```rst
:cylc:conf:`flow.cylc[runtime][<namespace>][meta]URL`
```